doc/cookbook/source/examples/classifier/multiclass_logisticregression.rst

```diff
@@ -0,0 +1,51 @@
+===============================
+Multi-class Logistic Regression
+===============================
+
+Logistic regression is a binary classifier that assigns class probabilities as: 
+
+.. math::
+ 
+    \mathrm{z} = \alpha +  \mathbf{\beta x} = \alpha + \beta_{1}x_{1} + ... + \beta_{d}x_{d} = \mathbf{\theta}^{T}\mathbf{x}
```

Comment:
See [equation 17](http://yelab.net/software/SLEP/manual.pdf) for the actual loss minimised.
You can translate that to this problem.